source/_integrations/plugwise.markdown

```diff
@@ -236,33 +240,44 @@ script:
           preset_mode: asleep
 ```
 
-### Supported devices
+## Supported devices
+
+The Plugwise integration relies on the [plugwise](https://pypi.org/project/plugwise/) module for Python. It currently provides support for:
```

Comment:
There is no reason the link end-user into libraries here. It doesn't add anything and merely may cause a technical scare. Let's remove it.